lib/packages_file.dart

```diff
@@ -128,6 +133,21 @@ void write(StringSink output, Map<String, Uri> packageMapping,
   }
 
   packageMapping.forEach((String packageName, Uri uri) {
+    // If [packageName] is empty then [uri] is the _default package name_.
+    if (allowDefaultPackage && packageName.isEmpty) {
+      final defaultPackageName = uri.toString();
```

Comment:
Don't bother using `final` for local variables. Just use `var`.

That's just personal style, but it's my package :grin:.

(You can let it stay, I'll just remove it later).